COMETwebapp/COMETwebapp/Components/Home.razor

```diff
@@ -0,0 +1,13 @@
+﻿<div style="text-align:center">
```

Comment:
Please add File header. Maybe discuss with @samatrhea what it should look like in Blazor/Razor.